src/LinuxTracing/LinuxTracingIntegrationTestPuppet.h

```diff
@@ -13,10 +13,12 @@ struct LinuxTracingIntegrationTestPuppetConstants {
   LinuxTracingIntegrationTestPuppetConstants() = delete;
 
   constexpr static uint64_t kSleepCount = 1000;
+  constexpr static const char* kNewName = "New Name";
   constexpr static const char* kSharedObjectFileName =
       "libLinuxTracingIntegrationTestPuppetSharedObject.so";
 
   constexpr static const char* kSleepCommand = "sleep";
+  constexpr static const char* kSetnameCommand = "setname";
```

Comment:
also maybe this set(main)threadname

Reply:
The variable or the string? The idea is to kind of match the calls they make: `pthread_setname_np` -> `setname`.

Reply:
maybe then align value with the name of libc function? so pthread_setname_np in this case? setname seems too generic.

Reply:
Done.